docs/README.md

```diff
@@ -2,27 +2,27 @@
 
 ## What is Feast?
 
-Feast \(**Fea**ture **St**ore\) is an operational data system for managing and serving machine learning features to models in production.
+Feast \(**Fea**ture **St**ore\) is an operational data system for managing and serving machine learning features to models in production. It can serve features from a low-latency offline store \(for real-time prediction\) or from an off-line store \(for scale-out batch scoring or training models\).
```

Comment:
```
Feast \(**Fea**ture **St**ore\) is an operational data system for managing and serving machine learning features to models in production. It can serve features from a low-latency online store \(for real-time prediction\) or from an off-line store \(for scale-out batch scoring or training models\).
```

Reply:
will address this in a followup PR since this is from a separate merge